<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserApplicationsSettingsWidget</class>
 <widget class="QWidget" name="UserApplicationsSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>581</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="langBox">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="langCombo"/>
      </item>
      <item>
       <widget class="QToolButton" name="langButton">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Style</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Color theme</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QComboBox" name="styleCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="colorThemeCombo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="projectBox">
     <property name="title">
      <string>Project</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="autoOpenProjectBox">
        <property name="text">
         <string>Open last project at startup</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QComboBox" name="askToSaveProject"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enableStatisticsEdit">
     <property name="toolTip">
      <string>Share now many times different tools were run or UI actions were activated. No user data is sent to UGENE.</string>
     </property>
     <property name="text">
      <string>Share usage metrics with UGENE team</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="updatesCheckBox">
     <property name="toolTip">
      <string>Do not check and ask about UGENE updates during UGENE startup.</string>
     </property>
     <property name="text">
      <string>Check for updates when UGENE starts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="highDpiAutoScalingCheckBox">
     <property name="toolTip">
      <string>Enable scaling for high-resolution (HiDPI) displays. Improves UI clarity on 4K and Retina screens. Requires restart.</string>
     </property>
     <property name="text">
      <string>Enable auto scaling in HDPI mode (requires restart)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="experimentsCheckBox">
     <property name="toolTip">
      <string>Enables experimental features hidden by default. May require restart for some features to be activated.</string>
     </property>
     <property name="text">
      <string>Enable experimental features</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="resetSettingsBox">
     <property name="toolTip">
      <string>Resets all UGENE settings to default. Requires restart.</string>
     </property>
     <property name="text">
      <string>Reset settings to default on the next run</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
